source/module_cell/read_pp.cpp

```diff
@@ -487,3 +487,181 @@ void Pseudopot_upf::setqfnew(const int& nqf,
         rho[ir] *= pow(r[ir], l + n);
     }
 }
+
+void Pseudopot_upf::complete_default(Atom_pseudo& pp)
+{
+    ModuleBase::TITLE("Pseudopot_upf", "complete_default");
+
+    // call subroutines
+    this->complete_default_h(pp);
+	this->complete_default_atom(pp);
+	this->complete_default_vl(pp);
+
+	if (pp.nbeta == 0) {
```

Comment:
It's a bit dangerous that this file contains more than 500 lines, is it possible to split it into more files?

Reply:
I have moved the functions corresponding to `complete_default` to a new file `read_pp_complete.cpp`.